### ISSUE TITLE
AllAnime Links Issues

### ISSUE THREAD

**Issue by LyuboA31** [aniyomiorg/aniyomi-extensions#2112]
### Source information

AllAnime 13.28

### Source language

English

### Steps to reproduce

1. Browse AllAnime
2. Enter an Anime Entry
3. Start an Episode

### Expected behavior

To show All Links Available on the AllAnime website for said Anime episode

### Actual behavior

Only mp4upload links are working All other links don't work and don't even show OK.ru shows up but it's Not working too

### Aniyomi version

Preview r6670

### Android version

Android 10

### Other details

All Hosts are Enabled in AllAnime extension settings 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.12.3.10](https://github.com/aniyomiorg/aniyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://aniyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/aniyomiorg/aniyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by LyuboA31** [comment]
> ### Source information
> AllAnime 13.27
> 
> ### Source language
> English
> 
> ### Steps to reproduce
> 1. Browse AllAnime
> 2. Enter an Anime Entry
> 3. Start an Episode
> 
> ### Expected behavior
> To show All Links Available on the AllAnime website for said Anime episode
> 
> ### Actual behavior
> Only shows up to 2 or 3 links out of 10 and even more on the Allanime website
> 
> ### Aniyomi version
> 0.12.3.10 and Preview r6151
> 
> ### Android version
> Android 10
> 
> ### Other details
> All Hosts are Enabled in AllAnime extension settings
> 
> ### Acknowledgements
> * [x]  I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
> * [x]  I have written a short but informative title.
> * [x]  I have updated the app to version **[0.12.3.10](https://github.com/aniyomiorg/aniyomi/releases/latest)**.
> * [x]  I have updated all installed extensions.
> * [x]  I have tried the [troubleshooting guide](https://aniyomi.org/help/guides/troubleshooting/).
> * [x]  If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/aniyomiorg/aniyomi/issues/new/choose).
> * [x]  I will fill out all of the requested information in this form.

**Comment by LyuboA31** [comment]
AllAnime 13.27 update still Only show 2 sources in Aniyomi while on AllAnime site there much more source you can check the issue on Doupo Cangqiong

**Comment by Rakeshyadav-19** [comment]
Right now for me not even 1 link is showing, ver. 13.27

**Comment by LittleFreak** [comment]
Im using v13.27 too, but almost every anime worked for me.

Some anime appear seem to be removed [Yumemiru Danshi wa Genjitsushugisha](https://allanime.to/anime/HcvmvD4jqYeLTGsZt/yumemiru-danshi-wa-genjitsushugisha) f. Example.
This one is listed with the addon, but due to the not existing entries at the website, it's empty.

Unlike you, I've also disabled a few streaming hoster, since they're blocked by my pihole anyway.
Maybe some are also blocked by your ISP? You can check this out in webview. 

Have you already tried to change the allanime TLD in the settings?

**Comment by LittleFreak** [comment]
NVM...
The Extension does not prioritize DUBS over SUBS, it blocks the other part entirely from loading.
You can check this out by changing the preferred setting.
